content/posts/July-2020.md

```diff
@@ -18,3 +18,18 @@ draft: false
 * You are now able to override configuration for builds, releases and other jobs (by [@TomasTomecek](https://github.com/TomasTomecek), [#676](https://github.com/packit-service/packit-service/pull/676)).
  Read more about this feature [here](/docs/configuration/#overriding-global-parameters).
 
+
+## Week 28 (July 6th - July 10th)
+
+### packit
+
+* Command `validate-config` is now working properly and not producing odd error messages, by [@TomasTomecek](https://github.com/TomasTomecek) and [@csomh](https://github.com/csomh), [#894](https://github.com/packit-service/packit/pull/894) and [#895](https://github.com/packit-service/packit/pull/895)
+* You will now properly see git-describe metadata when using `srpm` command in the %release, by [@TomasTomecek](https://github.com/TomasTomecek), [#894](https://github.com/packit-service/packit/pull/894)
```

Comment:
I don't understand what the `in the %release` means, but that might just be me :-)

Reply:
oki, will improve